RiotSwiftUI/Modules/UserSessions/UserSessionOverview/Coordinator/UserSessionOverviewCoordinator.swift

```diff
@@ -45,6 +45,8 @@ final class UserSessionOverviewCoordinator: Coordinator, Presentable {
         viewModel = UserSessionOverviewViewModel(sessionInfo: parameters.sessionInfo, service: service)
         
         hostingController = VectorHostingController(rootView: UserSessionOverview(viewModel: viewModel.context))
+        hostingController.vc_setLargeTitleDisplayMode(.never)
+        hostingController.vc_removeBackTitle()
```

Comment:
Unrelated, but the title animation looked whack and I don't think it was an intentional style to use anyway.